crates/sovereign-sdk/rollup-interface/src/fork/manager.rs

```diff
@@ -15,6 +15,7 @@ impl ForkManager {
         // Make sure the list of specs is sorted by the block number at which they activate.
         forks.sort_by_key(|fork| fork.activation_height);
 
+        // TODO: binary search here and assume forks is always sorted
```

Comment:
It is sorted above

Reply:
no I meant trust input is sorted no need to sort